app/models/concerns/bulkrax/errored_entries.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
```

Comment:
moved this stuff to a module so Rubocop would leave me alone about the CsvParser class length